api/src/main/java/uk/gov/hmcts/fees2/register/api/controllers/mapper/FeeDtoMapper.java

```diff
@@ -307,19 +336,6 @@ private void fillChannelType(Fee fee, String channel) {
     }
 
     /* --- */
-
-    private void fillCode(Fee fee, String code) {
```

Comment:
I'm so happy this is going away :grinning:

Reply:
This should be not have been here actually.